builds/misc/images.yaml

```diff
@@ -180,6 +180,13 @@ jobs:
           imageName: azureiotedge-c2dmessage-tester
           project: CloudToDeviceMessageTester
 
+     # Azure Monitor
+      - template: templates/image-linux.yaml
+        parameters:
+          name: Metrics Collector
```

Comment:
Recommend changing this name as there is currently a name clash in the build pipeline where we have two different images being built with the same named devops step.

Reply:
@and-rewsmith Are you referring to a clash in the template `name` parameter? I'm asking because that's a good change to make but not critical (won't cause an error in the build pipeline) so just clarifying...

Reply:
Renamed.